Doctor.js

```diff
@@ -0,0 +1,22 @@
+const Unit = require('./Unit');
+
+function Doctor(health, name){
+    Unit.apply(this, arguments);
+    this._healPower = 10;
+}
+
+Doctor.prototype = Object.create(Unit.prototype);
+
+Doctor.prototype.heal = function(Unit) {

```

Comment:
Its not a good idea to use capitalized variable names, it looks like you calling static methods of the constructor